src/cijoe/cli/cli.py

```diff
@@ -351,27 +352,33 @@ def cli_workflow(args):
     return err
 
 
-def cli_interface(path):
-    path = Path(path)
-    args = parse_args()
+def create_tmp_workflow(args, paths):
+    resources = get_resources()
+    paths = map(Path, paths)
+
+    template_path = resources["templates"]["core.example-tmp-workflow.yaml"].path
```

Comment:
Let's call it ``adhoc`` instead of **tmp**, since **tmp** is such as  loaded term...